src/Engine/GUI/Workspaces/OrbitalWorkspace.cpp

```
@@ -1432,172 +1540,289 @@ void OrbitalWorkspace::renderSceneResourceDetails() {
-			ImGui::BeginGroup();
+			ImGuiUtils::PushStyleClearButton();
 			{
-				if (ImGui::Button(ICON_FA_ARROW_ROTATE_LEFT)) {
-					m_codeEditor.Undo();
-				}
-				ImGuiUtils::CursorOnHover();
-				ImGuiUtils::TextTooltip(0, "Undo");
+				// Execution
+				ImGui::BeginGroup();
+				{
+					if (ImGui::Button(ICON_FA_PLAY)) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)